lsst/suit
# DM-8385: PDAC v1 deployment

https://jira.lsstcorp.org/browse/DM-8385
 - setup LSST/suit repo
 - added lsst logo onto the banner
 - make sure build scripts are fully functional
 - minor code refactoring

setup:
git clone https://github.com/lsst/suit
ensure both repos (firefly and suit) are in branch 'DM-8385_pdac_v1'

Things to test:
- able to build and deploy suit
`gradle :suit:buildAndDeploy`
- ensure dev mode works.  webpack should monitor firefly's source code as well as suit.
`gradle :suit:dev` 
- verify that LSST logo is used and that suit application is working as expected.  This require vpn into ncsa.
